src/main/java/duke/main/UI.java

```diff
@@ -0,0 +1,73 @@
+package duke.main;
+
+import duke.exceptions.DukeException;
+import duke.tasks.Task;
+import java.util.ArrayList;
+
+public class UI {
+    public void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public void greetUser(String chatBotName) {
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        printLine();
+        System.out.println("Hello! I'm " + chatBotName + "! SUIIII!!!");
+        System.out.println("What can I do for you?");
+    }
+
+    public void displayList(ArrayList<Task> list, int numOfTasks) {
+        printLine();
+        System.out.println("Here are the duke.tasks in your list:");
+        for (int i = 0; i < numOfTasks; i++) {
+            System.out.println((i + 1) + ". " + list.get(i));
```

Comment:
I think it might be better to get the number of tasks by list.size() instead of passing in an additional parameter.